### PR TITLE
NhatTruyen: Update the domain

### LINKED ISSUE #4173: NHATTRUYEN URL change (Nhattruyenup.com -> Nhattruyenvn.com)
### Source information

Nhattruyen 1.4.23

### Source language

Vietnamese - Tiếng Việt

### Source new URL

https://nhattruyenvn.com/

### Other details

Nhattruyen URL change and this it not duplicate or something like that, this is new one, pls fix it.

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Nyarunaa**
Closed #4173
Checklist: 

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by Norsze** [comment]
Link to an issue in your PR so we can see what you are trying to solve

**Comment by Nyarunaa** [comment]
> Link to an issue in your PR so we can see what you are trying to solve

I new in here, how i can link to issue ?

File: src/vi/nhattruyen/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ ext {
     extName = 'NhatTruyen'
     extClass = '.NhatTruyen'
     themePkg = 'wpcomics'
-    baseUrl = 'https://nhattruyenup.com'
+    baseUrl = 'https://nhattruyenvn.com'
     overrideVersionCode = 17
```
Comment (Smol-Ame) [inline src/vi/nhattruyen/build.gradle]: You need to bump versionCode too

Comment (Nyarunaa) [reply src/vi/nhattruyen/build.gradle]: i just update it

**Comment by Norsze** [comment]
> > Link to an issue in your PR so we can see what you are trying to solve
> 
> I new in here, how i can link to issue ?

You write the number of the issue after a # (so like #number) in the pull request up top before the checklist

**Comment by Nyarunaa** [comment]
> > > Link to an issue in your PR so we can see what you are trying to solve
> > 
> > 
> > I new in here, how i can link to issue ?
> 
> You write the number of the issue after a # (so like #number) in the pull request up top before the checklist

Thank for helping me!!!

**Comment by Nyarunaa** [comment]
Now should I just wait or do I need to do anything else?

**Comment by Norsze** [comment]
Question, did you install Android Studio to do this? Or did you do this in the GitHub editor? Did you test that it works?

**Comment by Nyarunaa** [comment]
> Question, did you install Android Studio to do this? Or did you do this in the GitHub editor? Did you test that it works?

my friend help me with this, he using some thing to test and is work, i just up file to github.

**Comment by Nyarunaa** [comment]
My friend just check, everthing is work, he just check by Android Studio.

**Review by Nyarunaa — COMMENTED** [review]
NhatTruyen: Update the domain

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**